### PR TITLE
SWC-6062: export StandaloneQueryWrapper

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/containers/table/StandaloneQueryWrapper.tsx [hunk 1]
```diff
@@ -136,52 +136,46 @@ const StandaloneQueryWrapper: React.FunctionComponent<StandaloneQueryWrapperProp
 
                     return (
                       <>
-                        {title ? (
```
Comment (jay-hodgson) [inline src/lib/containers/table/StandaloneQueryWrapper.tsx:139]: Removed logic that when a title was not supplied, then nothing is shown.  Verified in portals project that all configurations of StandaloneQueryWrapper give a title (so this should have no impact on portals).

File: src/lib/style/base/_core.scss [hunk 2]
```diff
@@ -1236,25 +1236,42 @@ hr ~ .SRC-wrapper {
...
-  .bootstrap-4-backport a:not(.btn),
-  .markdown td a:not(.btn),
-  .markdown a.SRC-userCard:not(.btn),
-  .secondary-links a:not(.btn),
-  .markdown .secondary-links a:not(.btn) {
-    @include SRC_Mixins.normalLink;
-    &:visited,
-    &:active,
-    &:link {
+  a,
```
Comment (jay-hodgson) [inline src/lib/style/base/_core.scss:1239]: Since all of these have ":not(.btn)", pushed that down a level

File: src/lib/style/base/_core.scss [hunk 3]
```diff
@@ -1236,25 +1236,42 @@ hr ~ .SRC-wrapper {
...
       @include SRC_Mixins.normalLink;
+      &:visited,
+      &:active,
+      &:link {
+        @include SRC_Mixins.normalLink;
+      }
+    }
+  }
+
+  .modal-body .dropdown-menu a {
```
Comment (jay-hodgson) [inline src/lib/style/base/_core.scss:1256]: Now that we style links in modal-body, this messes up anchors that are being used as dropdown menu items!  This is to correct that issue.

File: src/lib/style/base/_core.scss [hunk 4]
```diff
@@ -1236,25 +1236,42 @@ hr ~ .SRC-wrapper {
...
+      }
     }
   }
 
   .markdown a,
   .SRC-userCardName a,
   a.highlight-link,
   details > summary,
-  .modal-body a:not(.btn) {
+  .modal-body a {
```
Comment (jay-hodgson) [inline src/lib/style/base/_core.scss:1274]: Had an extra ":not(.btn)" here

File: src/lib/umd.index.ts [hunk 5]
```diff
@@ -21,6 +21,7 @@ import FavoritesPage from './containers/FavoritesPage'
 import { EvaluationEditorPage } from './containers/evaluation_queues/EvaluationEditorPage'
 import { EntityFinder } from './containers/entity_finder/EntityFinder'
 import QueryWrapperPlotNav from './containers/query_wrapper_plot_nav/QueryWrapperPlotNav'
+import StandaloneQueryWrapper from './containers/table/StandaloneQueryWrapper'
```
Comment (jay-hodgson) [inline src/lib/umd.index.ts:24]: Export!

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "synapse-react-client",
-  "version": "3.0.8",
+  "version": "3.0.9",
```
Comment (jay-hodgson) [inline package.json:3]: Bump SRC version to prep for release

**Review by nickgros — COMMENTED** [review]
LGTM

**Pull request merged**